lib/ABI/AArch64/AAPCS64.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+//
+// This file is distributed under the MIT Licence. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/StringRef.h"
+
+#include "revng/ABI/Definition.h"
+
+namespace abi::predefined::AArch64::AAPCS64 {
+
+constexpr llvm::StringRef Serialized = R"(---
```

Comment:
At this point put them in `share/revng/abi-definitions/aapcs64.yml`.
Use `ResourceFinder` to get the full path in a reliable way.

Reply:
Sure.

Reply:
1. Done.
2. There are at the same place they have always been: in `include/revng/Model/ABI.h`.

Reply:
Sounds good.

Reply:
On a second thought, they are OK where they are.
Sorry.

Reply:
You sure? I'd imagine `model::ABI::Values` going away soon. I guess we can move them then, but why the sudden change of mind?

P. S. There's also an option of duplicating them, not ideal, but will do for now.

Reply:
We can duplicate if you want, but I think the links are more useful in user-facing doc than in a YAML file that devs only look at. No strong opinion though, as you wish.

Reply:
I'm really afraid of duplicating to be honest. With the recent `yield::FunctionEdgeBase` fiasco being a good example why.

As for the user-facing part, the only concern is that it's going away. But I guess we can worry about it then. (I'm closing this thread)